### ISSUE TITLE
Persian  translation of the site

### ISSUE THREAD

**Issue by ghaseminya** [hexojs/site#418]
Hi
I am interested in providing a Persian translation for this site.
I add new locale in local repo then PR for change...

**Comment by NoahDragon** [comment]
We are welcome to any contribute on translation, please feel free to do so.

**Comment by ghaseminya** [comment]
Thanks @NoahDragon , I'm translating main site,
Just this 2 dir is needed for translate?
[docs](https://github.com/hexojs/site/tree/master/source/docs)
[api](https://github.com/hexojs/site/tree/master/source/api)
And this file:
[index.jade](https://github.com/hexojs/site/blob/master/source/index.jade)

**Comment by NoahDragon** [comment]
Correct, also you need to add the language into the theme/language folder https://github.com/hexojs/site/tree/master/themes/navy/languages
